src/role_scout.js

```diff
@@ -112,7 +112,12 @@ roles.scout.execute = function(creep) {
       // }
       return false;
     }
-
+    if (creep.isStuck()) {
+      creep.moveRandom();
+      creep.say('ImStuck', true);
+      creep.log(JSON.stringify(creep.memory.last) + ' ' + JSON.stringify(creep.isStuck()));
```

Comment:
If you want to add logging, it should be more descriptive, to find it in the code if you see it.

But I would remove it.